lib/Bridge/TolerantParser/SourceCodeFilesystem/ScfClassCompletor.php

```diff
@@ -88,15 +93,20 @@ public function complete(Node $node, string $source, int $offset): Response
         return Response::fromSuggestions($suggestions->sorted());
     }
 
-    private function isAlreadyImported(string $candidate, array $imports): bool
+    private function isImportNeeded($candidate, array $imports, ?string $currentNamespace): bool
     {
+        if ($currentNamespace === $candidate->namespace()) {
```

Comment:
What happens if this is the global namespace? (i.e. presumably `$currentNamespace` is `null`)

Reply:
indeed, ``$currentNamespace`` is ``null``. 

seems to work if the comparison is made non strict.